backend/tracim_backend/lib/proxy/proxy.py

```diff
@@ -6,6 +6,8 @@
 import requests
 from pyramid.response import Response as PyramidResponse
 from requests import Response as RequestsResponse
+from requests.auth import HTTPBasicAuth, AuthBase
+
 from tracim_backend.lib.utils.request import TracimRequest
 
 HOP_BY_HOP_HEADER_HTTP = (
```

Comment:
What is this HOP_BY_HOP?!

Reply:
Those are header which should not be forwarded by proxy, see also :
 fr : https://developer.mozilla.org/fr/docs/Web/HTTP/Headers
en : https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers

> These headers are meaningful only for a single transport-level connection and must not be retransmitted by proxies or cached

Reply:
just added a comment about that in code, with link to RFC : https://tools.ietf.org/html/rfc2616#page-92

Reply:
Perfect. I'm pretty sure developers who will maintain this code are even not knowing that there are headers named hop-by-hob - so a link to the documentation is perfect.